src/components/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import clsx from 'clsx';
+import Typography from '../Typography';
+import { Props } from './types';
+
+
+function Breadcrumb({
+    className,
+    divider,
+    crumbs,
+}: Props): ReactElement {
+
+
+    const crumbCount = crumbs.length - 1;
+    const classes = clsx(
+        'brew-Breadcrumb',
+        className,
+    );
+
+
+    return (
+        <div className={classes}>
+            {crumbs.map((crumb, index) => {
+                const notLast = crumbCount !== index;
+                return (
```

Comment:
Maybe add line spacing like we do regularly? 2 spaces before/after each section of content.